cps/tasks/metadata_extract.py

```diff
@@ -151,7 +151,7 @@ def _add_download_tasks_to_worker(self, requested_urls):
     def run(self, worker_thread):
         self.worker_thread = worker_thread
         log.info("Starting to fetch metadata for URL: %s", self.media_url)
-        self.start_time = self.end_time = datetime.now()
+        self.end_time = datetime.now(timezone.utc)
```

Comment:
Previously this was start_time, now it became end_time.

Reply:
@deldesir take a look, Thanks!

Reply:
Here I made this change to make the elapsed time start immediately when the task is called instead of when the worker thread is started. Not significant though, but more realistic IMHO.